src/data/plants/toxic.json

```diff
@@ -224,5 +220,93 @@
       "lat": "codiaeum"
     },
     "isSafe": false
+  },
+  {
+    "name": {
+      "pl": ["ciemiernik"],
+      "en": ["hellebore", "Christmas rose", "lenten rose", "Easter rose"],
+      "lat": "helleborus"
+    },
+    "isSafe": false
```

Comment:
if there was a choice between `true/false/wtf`, it would be the latter for most of the Christmas trees.

Reply:
I'm serious! :P